.travis.yml

```diff
@@ -13,16 +13,21 @@ virtualenv:
   system_site_packages: true
 
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  global:
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0"
+
+  matrix:
+  - LINT_CHECK="1"
+  - TESTS="1" ODOO_REPO="odoo/odoo"
```

Comment:
I'm not talking about translating doc, but modules themselves. Check https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml#L15